package.json

```diff
@@ -24,7 +24,7 @@
     "g:lint": "yarn workspaces foreach -ptv run lint --color",
     "g:lint-styles": "yarn workspaces foreach -ptv run lint-styles --color",
     "g:typecheck": "yarn workspaces foreach -ptv run typecheck",
-    "g:lint-staged-files": "lint-staged --allow-empty",
+    "g:lint-staged-files": "lint-staged --allow-empty --debug && yarn lint:dependency-versions",
```

Comment:
For some reason I couldnt make it to work? I changed the root package with `"next": "12.3.4"` then made a commit about it and expected it to fail but it didnt. 
Perhaps scripts should be left as they are but instead add a new rule into 
https://github.com/City-of-Helsinki/events-helsinki-monorepo/blob/main/packages/eslint-config-bases/lint-staged.config.js#L18
For example when any of the package.json or yarn.lock is changed then run the check

Reply:
Perhaps this isnt so easy I first though :) If it's too hard to get work then perhaps in another pr.

Reply:
Yeah it doesn't actually fail; it just shows the boxes with dep diff; so as last like after all the other output from lint-staged, there should be the "boxes" from dep check that just show that there's a difference.

Reply:
Actually there was a proablem on my machine and it actually works! So I can approve this now